module/VuFindTheme/tests/unit-tests/src/VuFindTest/View/Helper/HeadThemeResourcesTest.php

```diff
@@ -99,7 +99,7 @@ protected function getMockHeadMeta()
         $mock->expects($this->once())->method('prependHttpEquiv')
             ->with($this->equalTo('Content-Type'), $this->equalTo('text/html; charset=utf-8'));
         $mock->expects($this->once())->method('appendName')
-            ->with($this->equalTo('Generator'), $this->equalTo('fake-generator'));
+            ->with($this->equalTo(\Generator::class), $this->equalTo('fake-generator'));
```

Comment:
This is referring to the 'Generator' meta-tag, not the \Generator PHP class, so I don't think this is an appropriate change. I'll merge the other class constants, but I think we should revert this part.

Reply:
I agree, this is mistake

Reply:
Great, I'll just drop this commit the next time I rebase, since all the remaining parts have already been merged to dev.